editor/plugins/script_editor_plugin.h

```diff
@@ -255,6 +255,7 @@ class ScriptEditor : public PanelContainer {
 	enum ScriptSortBy {
 		SORT_BY_NAME,
 		SORT_BY_PATH,
+		SORT_BY_MODIFIED,
```

Comment:
```suggestion
		SORT_BY_MODIFIED_TIME,
```
To match `FilesystemDock`